reqirements.txt

```diff
@@ -0,0 +1,41 @@
+alembic==1.5.4
```

Comment:
You can try to shorten this list. I think that some dependencies from this list is extra and will be installed automatically with top-level dependencies.

Reply:
https://github.com/mstoro/nasa_api/pull/1/commits/dd4ea7f64b13c3b441a41ee185d61f81b7d20393